test-data/unit/check-modules.test

```diff
@@ -670,45 +670,45 @@ def f(x: str) -> None: pass
 [case testTypeCheckNamedModule]
 # cmd: mypy -m m.a
 [file m/__init__.py]
-None + 1
```

Comment:
It was just because in the builtins fixture `int.__radd__` isn't defined (only `int.__add__`). And the subtyping is only checked for the rhs, not for the lhs of these operators.